vite.config.js

```
@@ -4,15 +4,41 @@ import path from 'path';
-    }
-  })],
+  plugins: [
+    react({
+      babel: {
+        plugins: [["babel-plugin-react-compiler"]]
+      }
+    }),
+  ],
   resolve: {
     alias: {
       '@': path.resolve(__dirname, 'src'), // Use '@' as an alias for 'src'
```

ESLint

[error] '__dirname' is not defined.

(no-undef)